components/AccountBalanceCard/index.tsx

```diff
@@ -27,26 +27,37 @@ import useIconProps from '../../misc/useIconProps'
 interface AccountBalanceCardProps {
   account: Account
   accountBalance: AccountBalance
+  validators: Validator[]
   onVestingClick: () => void
   hideVestingButton?: boolean
 }
 
 const AccountBalanceCard: React.FC<AccountBalanceCardProps> = ({
   accountBalance,
   account,
+  validators,
   onVestingClick,
   hideVestingButton,
 }) => {
   const classes = useStyles()
   const { t, lang } = useTranslation('common')
   const theme: CustomTheme = useTheme()
   const { currency } = useGeneralContext()
+  const isValidator = validators.some((v) => v.address === account.address)
+  const data = isValidator
+    ? Object.keys(accountBalance.balance).map((k, i) => ({
+        name: t(k),
+        value: get(accountBalance, `balance.${k}.${account.crypto}.amount`, 0),
+        color: theme.palette.pieChart2[i % theme.palette.pieChart2.length],
+      }))
+    : Object.keys(accountBalance.balance)
+        .map((k, i) => ({
+          name: t(k),
+          value: get(accountBalance, `balance.${k}.${account.crypto}.amount`, 0),
+          color: theme.palette.pieChart2[i % theme.palette.pieChart2.length],
+        }))
+        .filter((v) => v.name !== 'Commissions')
```

Comment:
this filter won't work if user switches to other language. you can filter by the key `k` before mapping